### DISCUSSION TITLE
Dispatching to x-model

### DISCUSSION THREAD

**Discussion by osvik** [alpinejs/alpine#2822] [7. Bugs (archived)]
Hi. I've submitted this issue a few minutes ago but I don't see it anywhere. Sorry if it's a duplicate.

The "Dispatching to x-model" example in the documentation doesn't work.

**Comment by SimoTod** [comment]
It's in discussion tab just before this one. This project doesn't use the issue tab.
As a separate note, the example in the doc does work: https://codepen.io/SimoTod/pen/YzYvKpJ

**Comment by osvik** [comment]
Hi.
Thanks.
The span x-text line is not in the documentation and it confused me.
All the best,
Osvaldo 

Enviado do Yahoo Mail para Android 
 
  Em sábado, 9 abr., 2022 às 14:05, Simone ***@***.***> escreveu:   


It's in discussion tab just before this one. This project doesn't use the issue tab.
As a separate note, the example in the doc does work: https://codepen.io/SimoTod/pen/YzYvKpJ

**Reply by musamiyas** [reply]
Good evening sig osvik please have some programs to compare large text files 20 gigabytes
